spec/fixtures/config.yml

```diff
@@ -0,0 +1,20 @@
+# Dummy sample file for testing parsing of configuration file into PushmiPullyu
```

Comment:
Some of the files seem to be exact copy of the files in PMPY. could you please review which ones are actually needed?

Reply:
Used this file for testing:
spec/swift_ingest/ingestor_spec.rb
6:      sample_file = 'spec/fixtures/config.yml'
will modify it content to be more clear